martin_callejas/task1_tic-tac-toe/src/jalasoft/training/Game.java

```diff
@@ -59,8 +59,12 @@ public void play(){
             }
             strAvailableSpots += "]";
             System.out.println("Available positions: " + strAvailableSpots);
+
+            String strPosition;
             System.out.print("Position? ");
-            position = scanner.nextInt();
+            strPosition = scanner.next();
```

Comment:
why you dont use nextInt instead and take care of mismatch? just asking

Reply:
If a non-integer is entered, Scanner.nextInt() will throw an unhandled exception. So first I'm storing input in a String and then parsing with tryParse in the next line.